AUTHORS

```diff
@@ -276,6 +297,7 @@ Zang Li                                 zangli@google.com
 Zhiyuan Hou                             zhiyuan2048@linux.alibaba.com
 Zhu Yan                                 hackzhuyan@gmail.com
 Zinin D.A                               admin@kami-no.ru
+黄学森                                     huangxuesen@kuaishou.com
```

Comment:
The alignment is weirdly off here in the second column, but unsure if that's because I don't have the character set installed for this language on my machine (the author's name doesn't render)

Reply:
I see, I can probably get a romanization of this one.